.github/workflows/ci.yml

```diff
@@ -38,7 +38,9 @@ jobs:
     - name: Setup tests
       working-directory: ${{runner.workspace}}/libmcleece
       shell: bash
-      run: (cd dist/bin/ && (echo "password" | ./mcleececli keypair --key-path=/tmp/test))
+      run: |
+         (cd dist/bin/ && (echo "password" | ./mcleececli keypair -s --key-path=/tmp/test))
+         (cd dist/bin/ && (echo "password" | ./mcleececli keypair --key-path=/tmp/cbox))
```

Comment:
`simple` and `cbox` keypairs are currently treated as entirely distinct. In reality, the latter is a superset of the former (a `cbox` private key is `x25519 + simple`, and likewise for the public key), but I'm not currently supporting mixing and matching.